### PR TITLE
Locatie/#198 locatie overzicht opvragen

### PR DESCRIPTION
**Pull request by SjorsScholten**
- added table component, can be used by other views as a generic table component
- added location overview component
- added location overview page, and added location overview and back button to the page
- added routing to the location overview page and connected the menu button to route to the page
- added location class
- added call to the api gateway to retrieve locations

known issues:
- api call is still commented in location overview table, when api gateway is available uncomment and test to see if its working
- for the sorting to work, the table headers need to be the same as the name for the properties in the table items

### CODE REVIEW

**Comment by ShadyDL** [comment]
Ga er nu naar kijken

File: src/classes/Location.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export default class Location {
+    public Id: Number;
+    public Stad: String;
```
Comment (ShadyDL) [inline src/classes/Location.ts:3]: Change property names to English to match backend model
- City
- Location
- CollectionPoint

File: src/classes/Location.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    public Locatie: String;
+    public Afhaalpunt: String;
+
+    constructor(id: Number, stad: String, locatie: String, afhaalpunt: String){ 
+        this.Id = id;
+        this.Stad = stad;
+        this.Locatie = locatie;
```
Comment (ShadyDL) [inline src/classes/Location.ts:10]: See previous

File: src/views/LocatieOverzicht.vue [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<template>
+    <div>
```
Comment (ShadyDL) [inline src/views/LocatieOverzicht.vue:2]: Change file name to LocationOverview.vue

**Comment by SjorsScholten** [comment]
naamgeving is geupdate naar het engels

**Review by ShadyDL — APPROVED** [review]

**Pull request merged**